app/src/main/res/values-land/dimens.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <dimen name="home_padding_end">72dp</dimen>
-  <dimen name="home_padding_start">72dp</dimen>
+  <dimen name="home_padding_start">65dp</dimen>
```

Comment:
According to [Working on UI](https://github.com/oppia/oppia-android/wiki/Working-on-UI#paddingmarginview-dimensions) we recommend using dimensions which are multiples of 4

Reply:
@Akshat-gour Conversation should be resolved only by reviewers or the comment author.

Reply:
Done

Reply:
as per here as the comment author is @FareesHussain , so he can confirm the conversation and look for what changes suggested and mark the conversation resolved. 

The PR author should not mark the conversation resolved, as you @Akshat-gour are the PR author here.

Reply:
@FareesHussain 
please review it. I made changes as suggested

Reply:
You can mark this resolved now.